### PR TITLE
Say something when tazer lacks charges.

### PR DESCRIPTION
**Pull request by woutershep**
And since ::tazer now does it ::tazer2 can use that too.

### CODE REVIEW

File: src/iuse.cpp [hunk 1]
```diff
@@ -4041,6 +4041,7 @@ int iuse::portal(player *p, item *it, bool, const tripoint& )
 int iuse::tazer(player *p, item *it, bool, const tripoint &pos )
 {
     if( !it->ammo_sufficient() ) {
+        p->add_msg_if_player( m_info, _("Insufficient power") );
```
Comment (mugling) [inline src/iuse.cpp]: * astyle
* `"Insufficient power to use your %s", it->tname().c_str()`

**Comment by woutershep** [comment]
Hmmm i found out testing the changes its only a UPS modded tazer that stays silent. I'm not sure how to proceed from here. The 'normal' tazer says 'Your tazer has X charge but needs Y.' but the UPS modded tazer says nothing. Is it a known problem that UPS modded devices don't have the default error? It's tempting to close this PR and open a bug for the other instead since this looks like symptom patching instead of the underlying thing.

**Comment by woutershep** [comment]
I think player::has_enough_charges hasn't been updated for the gradual UPS recharging vs consuming directly from the UPS.

**Comment by mugling** [comment]
> It's tempting to close this PR and open a bug for the other instead since this looks like symptom patching instead of the underlying thing.

**Pull request closed without merging**